script.js

```diff
@@ -0,0 +1,137 @@
+// ---------- functions to open and close the hamburger men-------//
+/* eslint-disable */
```

Comment:
- [ ] [Optional] I noticed that you disabled the linters in the code, and I want to emphasize that they are important. A linter is a valuable tool that analyzes your source code to identify programming errors, bugs, stylistic issues, and potentially problematic constructs (source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))). Please remove the eslint disable and take the time to address the linter errors. It is essential for maintaining code quality and ensuring a more reliable and consistent codebase. Thank you.